.travis

```diff
@@ -0,0 +1,16 @@
+language: node_js
```

Comment:
Does this need to be named `.travis.yml`?

Reply:
Fixed.